### ISSUE TITLE
Unable to login using Chrome

### ISSUE THREAD

**Issue by maristane** [HabitRPG/habitica#6453]
I just hit the button and nothing happens.
Then when I open the console this is what I get:

```
SyntaxError: Unexpected token c
    at Object.parse (native)
    at Object.<anonymous> (https://habitica.com/static-3df77c77.js:24:13037)
    at Object.i [as invoke] (https://habitica.com/static-3df77c77.js:12:23944)
    at Object.c [as $get] (https://habitica.com/static-3df77c77.js:12:22340)
    at Object.i [as invoke] (https://habitica.com/static-3df77c77.js:12:23944)
    at https://habitica.com/static-3df77c77.js:12:21990
    at g (https://habitica.com/static-3df77c77.js:12:23577)
    at Object.i [as invoke] (https://habitica.com/static-3df77c77.js:12:23911)
    at D.instance (https://habitica.com/static-3df77c77.js:13:15544)
    at https://habitica.com/static-3df77c77.js:13:9876(anonymous function) @ static-3df77c77.js:13
```

On Firefox it works just fine. Since the website is probably working well with other Chrome users, it may have something to do with my browser extensions or my antivirus. But It's not impossible that other users may be experiencing the same problem, so I decided to open this issue.

Does anybody have any suggestion on what should I try to do?

**Comment by maristane** [comment]
Well, that was quick.
I tested on anonymous mode in Chrome and it logged in, so I guessed it had something to do with the cookies. I then deleted any cookie related to habitica/habitrpg and it worked again.

I was having this problem since the HabitRPG -> Habitica transition, but I've stopped using the website at that time and just recently decided to come back. 

So, if any old user is experiencing the same problem, just remove the old cookies (instructions here: https://support.google.com/chrome/answer/95647).
